packages/react/src/PricingOptions/PricingOptions.tsx

```diff
@@ -0,0 +1,479 @@
+  ({children, className, 'data-testid': testId, ...rest}, ref) => {
+    return (
+      <Heading
+        as="h4"
+        className={clsx(styles['PricingOptions__feature-list-heading'], className)}
+        data-testid={testId || testIds.featureListHeading}
+        ref={ref}
+        size="subhead-medium"
+        {...rest}
+      >
+        {children}
+      </Heading>
+    )
+  },
+)
+
+type PricingOptionsFeatureListItemProps = PropsWithChildren<BaseProps<HTMLLIElement>> & {
+  'data-testid'?: string
+  variant?: 'included' | 'excluded'
+}
+
+const PricingOptionsFeatureListItem = forwardRef<HTMLLIElement, PricingOptionsFeatureListItemProps>(
+  ({children, className, variant = 'included', 'data-testid': testId, ...rest}, ref) => {
+    const itemLeadingVisual = variant === 'included' ? CheckIcon : XIcon
+    const itemLeadingVisualAriaLabel = variant === 'included' ? 'Includes' : 'Does not include'
+    const itemLeadingVisualFill =
+      variant === 'included' ? 'var(--brand-color-accent-primary)' : 'var(--brand-color-text-muted'
+
+    return (
+      <UnorderedList.Item
+        className={clsx(
+          styles['PricingOptions__feature-list-item'],
+          {[styles['PricingOptions__feature-list-item--excluded']]: variant === 'excluded'},
+          className,
+        )}
+        data-testid={testId || testIds.featureListItem}
+        leadingVisual={itemLeadingVisual}
+        leadingVisualAriaLabel={itemLeadingVisualAriaLabel}
+        leadingVisualFill={itemLeadingVisualFill}
+        ref={ref}
+        {...rest}
+      >
+        {children}
+      </UnorderedList.Item>
+    )
+  },
+)
+
+type RestrictedPolymorphism =
+  | (BaseProps<HTMLAnchorElement> & {as?: 'a'})
+  | (BaseProps<HTMLButtonElement> & {as?: 'button'})
+
+type PricingOptionsActionsProps = {
+  as?: 'a' | 'button'
+  href: string
+  'data-testid'?: string
+} & ButtonBaseProps &
+  RestrictedPolymorphism
+
+const PricingOptionsPrimaryAction = forwardRef<
+  HTMLAnchorElement | HTMLButtonElement,
+  PropsWithChildren<PricingOptionsActionsProps>
+>(({as, children, className, 'data-testid': testId, ...rest}, ref) => {
+  return (
+    <Button
+      as={as}
+      data-testid={testId || testIds.primaryAction}
+      className={clsx(styles['PricingOptions__primary-action'], className)}
+      size="medium"
+      variant="primary"
+      {...rest}
+      ref={ref as React.Ref<HTMLButtonElement>}
+      block
+    >
+      {children}
+    </Button>
+  )
+})
+
+const PricingOptionsSecondaryAction = forwardRef<
+  HTMLAnchorElement | HTMLButtonElement,
+  PropsWithChildren<PricingOptionsActionsProps>
+>(({as, children, className, 'data-testid': testId, ...rest}, ref) => {
+  return (
+    <Button
+      as={as}
+      className={clsx(styles['PricingOptions__primary-action'], className)}
+      data-testid={testId || testIds.secondaryAction}
+      variant="secondary"
+      {...rest}
+      ref={ref as React.Ref<HTMLButtonElement>}
+      block
```

Comment:
Same comment as above..

```suggestion
      ref={ref as React.Ref<HTMLButtonElement>}
      block
      {...rest}
```